save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -508,6 +508,7 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
                 }
                 setState {
                     isUploading = false
+                    window.location.reload()
```

Comment:
Why do you want to use it here? I am sure you can rerender the required part of a page instead of rendering the whole page for the second time

Reply:
When changing the avatar, we still have the same object with the same name. But the content of the bytearray is changing. Therefore, you need to re-render the page to get a new avatar.

Reply:
Oh, I see
But what I suggest is to use some other way to re-render the page (only avatar). Thus there will be no useless re-render. Probably it might be a good idea to add FC wrapper for avatar so it could
1. re-render
2. display better tooltips

Can be done in next PR but if so then do it right after this one

Reply:
OK. I will do it in the next PR. I won't refactor for now

Reply:
Then please open a new issue and mention there both things I wrote above